indy_common/test/auth/test_auth_actions.py

```diff
@@ -0,0 +1,17 @@
+from indy_common.auth_actions import compile_action_id, ADD_PREFIX, EDIT_PREFIX
+
```

Comment:
Please add tests for `split_action_id`. 
`split_action_id(compile_action_id(action)) == action??`

Reply:
done